pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+markers =
```

Comment:
Where is the code for assigning resources based on markers?

Reply:
This is not dealt with here. This PR is for making it easier to filter tests when resources are not available.